CHANGELOG.md

```diff
@@ -82,6 +82,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: It is now logical to access the Chain Reaction Device after ending the Chain Reaction sequence. This also means climbing the Chain Reaction Room later has slightly fewer requirements.
 - Changed: Using the Stand On Frozen Enemies trick to get Missile Tank 1 in Melee Tutorial Room now requires Highly Dangerous Logic.
 - Changed: Fighting Corpius without tricks now requires 198 damage, down from 199. This makes the fight logical on Strict Damage strictness with 1 Energy Tank.
+- Changed: The connection in Arbitrary Enky Room directly from Total Recharge to Dock to Teleport to Dairon is now always logical when having Morph Ball.
```

Comment:
This isn't from the Total Recharge. It would be from the upper center, the Start Point node.